### DISCUSSION TITLE
Superb work, the progress of WinPaletter is impressive !

### DISCUSSION THREAD

**Discussion by OrthodoxWindows** [Abdelrhman-AK/WinPaletter#121] [General]
It's been a while since I've been active here, but I can only see the impressive progress of WInPaletter!
@Abdelrhman-AK The work you've done seems extensive. Congratulations.

For my part, I will try to help to translate the software into French and to indicate the references of the high contrast mode as soon as I have time.

**Comment by Abdelrhman-AK** [comment]
You're welcome :)
and thanks for your efforts!
